cmd/gangway/handlers.go

```diff
@@ -105,13 +102,13 @@ func generateKubeConfig(cfg *userInfo) clientcmdapi.Config {
 				Name: cfg.ClusterName,
 				Cluster: clientcmdapi.Cluster{
 					Server:                   cfg.APIServerURL,
-					CertificateAuthorityData: []byte(caData),
+					CertificateAuthorityData: []byte(cfg.ClusterCA),
 				},
 			},
 		},
 		Contexts: []clientcmdapi.NamedContext{
 			{
-				Name: cfg.Email,
+				Name: cfg.ClusterName,
```

Comment:
At one point the contextname was `user@clustername` to make them unique. Is this still the case?

Reply:
Based on #87 , I think the context name has always been ClusterName. `user@clustername` was used for AuthInfo name.